versioned_docs/version-0.5.0/release-notes/release-notes-for-kubeslice-oss-0.5.1.mdx

```diff
@@ -0,0 +1,50 @@
+# Release Notes for KubeSlice OSS 0.5.1
+
+*Release date: 06 Jan 2023*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New
+These release notes describe the new changes and enhancements in this version.
+
+### Network Service Mesh Upgrade 
```

Comment:
We need to remove the release notes that's related to NSM upgrade in v0.5.0 including known issues and K8s 1.24 and just retain the kubeslice-cli info. 

@mnmaha - can you confirm?

Reply:
@uma-kt  - NSM upgrade has already been released; remove it.

Reply:
done